### ISSUE TITLE
zcash address, which was displaying on profile, is no longer showing

### ISSUE THREAD

**Issue by grempe** [keybase/keybase-issues#2746]
I added a zcash address a few days ago. You can see it here in my sigchain.

https://keybase.io/grempe/sigchain#58e5a3365869cf3f6a245f605b5b48d59190a342f54cd650450baa28ad5640520f

Starting, I believe, today it still shows in my sigchain, but is no longer displayed on the user profile. Instead it says `set a Zcash address`

https://keybase.io/grempe

**Comment by maxtaco** [comment]
Looks like a bug, we'll take a look ASAP

@malgorithms -- yours appears to be gone too.

**Comment by maxtaco** [comment]
@grempe  I do see yours with the CLI

@mmaxim's is showing on the Web and the CLI.. something is going on here...

**Comment by grempe** [comment]
Yes, I see a zcash address for `mikem` in the web profile view and the sigchain page, and via CLI:

https://keybase.io/mikem

https://keybase.io/mikem/sigchain#77537cfbcd2c74a119b34af5c6d79efd565f4be78738c5280835158cbce33f030f

```
$ keybase id mikem
▶ INFO Identifying mikem
✔ public key fingerprint: DBAF 5AEF B0EB BA22 F181 40E5 B814 608B 7797 CA08
฿  bitcoin 1G3zXJV2seYgjN6i34GbCyrp9YMsp2v1ik
฿  zcash zcKJ7Zo4GXkDQwtTeiu2dbMuxqx4BwYKHmA2z1GwYL9hFhsYHGx6SZzjAdKNhnAQCSce2xYdqUeY1t5aEAn67yf4H3sjYf7
✔ admin of DNS zone mikemaxim.com: found TXT entry keybase-site-verification=mPvhUeOlTERv_YIu02sPwxm3aZ-9UFf4vQeoQCS-8hE
✔ admin of DNS zone mmaxim.org: found TXT entry keybase-site-verification=JcJiv15lZbvBTI4ldNJMlb68biFClvNefz9PP1nCy6o
✔ admin of DNS zone okcupid.com: found TXT entry keybase-site-verification=piRpEGDSgyYhnNK5eo7NdvZv13vHlSixQHoVL9MIPKI
✔ admin of DNS zone maxim.family: found TXT entry keybase-site-verification=-2OpcnFespJEWaeZBlplpz5NoZnoQg0rB27UDUCRUCY
✔ admin of DNS zone mikemax.im: found TXT entry keybase-site-verification=qDa8OIUmAEFE059770nTZdNALwlsJ88KjIz3SfGrY7Q
✔ admin of mmaxim.org via HTTP: http://mmaxim.org/keybase.txt
✔ "candrencil" on twitter: https://twitter.com/candrencil/status/436538176702660608
✔ "candrencil" on reddit: https://www.reddit.com/r/KeybaseProofs/comments/2dw0ko/my_keybase_proof_redditcandrencil_keybasemikem/
✔ "mike.maxim" on facebook: https://facebook.com/10153922233870382_10154017925505382
✔ "mmaxim" on github: https://gist.github.com/9117747
✔ "candrencil" on coinbase: https://coinbase.com/candrencil/public-key
```

But I do not see one for `chris` via CLI or WEB, but `chris` seems to not have added a zcash address according to his sigchain so that would be what I would expect. I'm looking at the right account right?

https://keybase.io/chris
https://keybase.io/chris/sigchain

```
/tmp$ keybase id chris
▶ INFO Identifying chris
✔ <tracked> public key fingerprint: 94AA 3A5B DBD4 0EA5 49CA BAF9 FBC0 7D6A 9701 6CB3
You last followed chris on 2016-03-22 13:39:55 PDT
฿  bitcoin 1BjgMvwVkpmmJ5HFGZ3L3H1G6fcKLNGT5h
✔ <tracked> admin of DNS zone chriscoyne.com: found TXT entry keybase-site-verification=2_UwxonS869gxbETQdXrKtIpmV1u8539FmGWLQiKdew
✔ <tracked> "malgorithms" on github: https://gist.github.com/2d5bed094c6429c63f21
✔ <tracked> "malgorithms" on reddit: https://www.reddit.com/r/KeybaseProofs/comments/4vt4ra/my_keybase_proof_redditmalgorithms_keybasechris/
✔ <new> "ccoyne77" on facebook: https://facebook.com/10158198173985206_10158632053400206
✔ <tracked> "bitcoyne" on coinbase: https://coinbase.com/bitcoyne/public-key
✔ <tracked> "malgorithms" on twitter: https://twitter.com/malgorithms/status/433640580220874754
✔ <tracked> admin of keybase.io via HTTPS: https://keybase.io/.well-known/keybase.txt
✔ <new> admin of chris.keybase.pub via HTTPS: https://chris.keybase.pub/keybase.txt
✔ <tracked> "malgorithms" on hackernews: https://news.ycombinator.com/user?id=malgorithms
✔ <new> admin of chriscoyne.com via HTTPS: https://chriscoyne.com/keybase.txt
```

For me `grempe` I see it via CLI, and in my sigchain on the web, but not in the web profile view. Just taking a stab at this, in the last 24 hours I revoked my eldest key (a PGP key I was transitioning from). I don't think the zcash key was directly signed by that key, but instead by my potion-factory device key (can't really tell since the graph doesn't show zcash or BTC proofs, which would be nice). But the other two users seem to be displaying as one would expect so I'm trying to think what may be different with my account.

See the revoked PGP key here.
https://keybase.io/grempe/graph

**Comment by grempe** [comment]
Aha, yes, the zcash entry was signed with my old PGP key (`A4A2 88A3 BECC AE17`). Why that and not my device key?

https://keybase.io/grempe/sigchain#58e5a3365869cf3f6a245f605b5b48d59190a342f54cd650450baa28ad5640520f

So maybe a display bug if the PGP key that signed a cryptocurrency address is revoked.

**Comment by grempe** [comment]
Going back through my zsh history it looks like I added the zcash address using the curl command (which attached it to PGP address). The BTC address was signed with a device key (using keybase CLI).

I think I did this in response to discussion with @malgorithms in this issue where it was also noted that there is an [lightly/un]documented difference in outcome if you use curl vs keybase CLI:

https://github.com/keybase/keybase-issues/issues/2735

**Comment by malgorithms** [comment]
> @malgorithms -- yours appears to be gone too.

IIRC I haven't added one.

**Comment by grempe** [comment]
Note, this issue still exists. What's interesting is that my local keybase client GUI just updated and I can see the zcash address in the GUI profile view, but I still cannot see the zcash address in the website profile view. See attached screenshot.

<img width="1337" alt="image 2017-01-20 at 12 34 04 pm" src="https://cloud.githubusercontent.com/assets/117/22164684/2648c4c0-df0d-11e6-9ef1-afddea9f02d7.png">

**Comment by cjb** [comment]
Thanks for the ping.  @maxtaco, @malgorithms?

**Comment by cjb** [comment]
I see @grempe's zcash address on a command line `keybase id grempe`, but not on https://keybase.io/grempe
